### ISSUE TITLE
'rails generate rspec:install' stuck in rails 4.1.0

### ISSUE THREAD

**Issue by kidlab** [rspec/rspec-rails#996]
I'm using:

```
ruby '2.1.1'

# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
gem 'rails', '4.1.0'

group :test, :development do
  gem 'debugger'
  gem 'rspec-rails', '~> 2.14.2'
  gem 'uuidtools'
end
#...
```

I encountered a problem that it got stuck when install rspec with this command:

```
rails generate rspec:install
```

No log, no error message at all! However, when I terminated the command there's an error:

```
/Users/manvuong/.rvm/gems/ruby-2.1.1/gems/spring-1.1.2/lib/spring/client/run.rb:47:in `gets': Interrupt
    from /Users/manvuong/.rvm/gems/ruby-2.1.1/gems/spring-1.1.2/lib/spring/client/run.rb:47:in `verify_server_version'
    from /Users/manvuong/.rvm/gems/ruby-2.1.1/gems/spring-1.1.2/lib/spring/client/run.rb:19:in `call'
    from /Users/manvuong/.rvm/gems/ruby-2.1.1/gems/spring-1.1.2/lib/spring/client/command.rb:7:in `call'
    from /Users/manvuong/.rvm/gems/ruby-2.1.1/gems/spring-1.1.2/lib/spring/client/rails.rb:23:in `call'
    from /Users/manvuong/.rvm/gems/ruby-2.1.1/gems/spring-1.1.2/lib/spring/client/command.rb:7:in `call'
    from /Users/manvuong/.rvm/gems/ruby-2.1.1/gems/spring-1.1.2/lib/spring/client.rb:26:in `run'
    from /Users/manvuong/.rvm/gems/ruby-2.1.1/gems/spring-1.1.2/bin/spring:48:in `<top (required)>'
    from /Users/manvuong/.rvm/gems/ruby-2.1.1/gems/spring-1.1.2/lib/spring/binstub.rb:11:in `load'
    from /Users/manvuong/.rvm/gems/ruby-2.1.1/gems/spring-1.1.2/lib/spring/binstub.rb:11:in `<top (required)>'
    from /Users/manvuong/Workspace/squar-ad-server/bin/spring:16:in `require'
    from /Users/manvuong/Workspace/squar-ad-server/bin/spring:16:in `<top (required)>'
    from bin/rails:3:in `load'
    from bin/rails:3:in `<main>
```

Any thought? Maybe this is a problem of https://github.com/rails/spring?

**Comment by seanfcarroll** [comment]
Worked for me also only after stopping spring

**Comment by PeterBloom** [comment]
I just ran `sprint stop` and things started working.

**Comment by adarsh** [comment]
Same here, but just a note for posterity that `sprint stop` is a typo and should be `spring stop`.

**Comment by cupakromer** [comment]
I'm going to lock this issue. Recent comments are merely confirming the issue resolution. If anyone runs into a similar issue not resolved accordingly, please open a new issue. :heart:
